stages/org.osbuild.dracut

```diff
@@ -10,6 +11,35 @@ def yesno(name: str, value: bool) -> str:
     return f"--{prefix}{name}"
 
 
+class DracutChroot:
```

Comment:
(super nitpick) it took me a moment to realize that this sets up the dirs but not doing the chroot()ing so I wonder if a slightly better name might be `DracutMounts` or `MountedDevProcSys` or something

Reply:
Sure, makes sense :+1:

Reply:
done